src/examples.jl

```diff
@@ -1165,24 +1165,19 @@ const _examples = PlotExample[
     ),
     PlotExample( # 55
         "3D axis flip / mirror",
-        "",
+        "An example showing how to flip or mirror axes in 3D, and use a custom layout",
         [
             :(
               begin
-		using LinearAlgebra
-		scalefontsizes()
+                using LinearAlgebra
                 scalefontsizes(.5)
 
-                x, y = -6:0.5:10, -8:0.5:8
-		function f(x,y)
-		  r = norm([x,y])
-		  sinc(r / pi)
-		end
+                x, y = collect(-6:0.5:10), collect(-8:0.5:8)
 
-                args = (x, y, f)
+                args = x, y, (x, y) -> sinc(norm([x, y]) / π)
                 kwargs = Dict(
                     :xlabel => "x", :ylabel => "y", :zlabel => "z",
-                    :grid => true, :minorgrid => true, :dpi => 200
+                    :grid => true, :minorgrid => true, :dpi => 300
```

Comment:
Not sure about the dpi in the docs, we'll see once it's fully rendered by `Documenter.jl`...